src/components/Json-results.js

```diff
@@ -1,36 +1,28 @@
-import React from 'react';
-import Intent from '../intent/json-intent';
+import React, { Component } from 'react';
 
-var JsonResults = React.createClass({
+export default class JsonResults extends Component {
```

Comment:
Since writing this I've gone for pure function style and kept away from ES6 class just didn't have time to update it.
Have a look at this article [here](https://medium.com/@housecor/react-stateless-functional-components-nine-wins-you-might-have-overlooked-997b0d933dbc#.o6ydma5hz)

```
const JsonResults = () => (....
```